client/blocks/reader-post-card/byline.jsx

```diff
@@ -79,7 +80,7 @@ class PostByline extends React.Component {
 							author={ post.author }
 							siteUrl={ streamUrl }
 							post={ post }>
-							{ post.author.name }
+							{ decodeEntities( post.author.name ) }
```

Comment:
I've just been playing with `he` (v0.50 of which we already include in package.json) and that successfully decodes it:

https://github.com/mathiasbynens/he

(Added in 85e9af7 for experiment's sake)